scripts/gen-docs/gen-docs.go

```diff
@@ -42,9 +62,10 @@ func NewCmdGenDocs(rootCmd *cobra.Command) *cobra.Command {
 }
 
 // runGenDocs generates the markdown files for the flag documentation
-func runGenDocs(rootCmd *cobra.Command, path *string) func(cmd *cobra.Command, args []string) {
-	rootCmd.DisableAutoGenTag = true
+func runGenDocs(path *string) func(cmd *cobra.Command, args []string) {
+	irgendwascmd := sparrowcmd.BuildCmd("")
+	irgendwascmd.DisableAutoGenTag = true
 	return func(cmd *cobra.Command, args []string) {
-		_ = doc.GenMarkdownTree(rootCmd, *path)
+		_ = doc.GenMarkdownTree(irgendwascmd, *path)
```

Comment:
I would check for the error, we should still check if something goes wrong during doc creation. `irgendwascmd` is funny and I do appreciate a funny name, but I think given the scope of the variable a simple `c` would be enough :)

Reply:
oh yeah forgot to change that :D